fdbclient/include/fdbclient/DatabaseContext.h

```diff
@@ -374,12 +374,18 @@ class DatabaseContext : public ReferenceCounted<DatabaseContext>, public FastAll
 	Future<OverlappingChangeFeedsInfo> getOverlappingChangeFeeds(KeyRangeRef ranges, Version minVersion);
 	Future<Void> popChangeFeedMutations(Key rangeID, Version version);
 
+	// BlobGranule API.
 	Future<Key> purgeBlobGranules(KeyRange keyRange,
 	                              Version purgeVersion,
 	                              Optional<TenantName> tenant,
 	                              bool force = false);
 	Future<Void> waitPurgeGranulesComplete(Key purgeKey);
 
+	Future<bool> blobbifyRange(KeyRange range);
+	Future<bool> unblobbifyRange(KeyRange range);
+	Future<Standalone<VectorRef<KeyRangeRef>>> listBlobbifiedRanges(KeyRange range, int rowLimit);
```

Comment:
I like having the row limit, but other paginated db calls usually have some sort of flag in their return indicating that the limit was hit and the result may be incomplete.
Just to clarify, is the expected semantic that the client must determine this and retry? For example, if i pass a limit of 100 and get 100 ranges back, is it my job to then call again with an updated key range to ensure the remaining result is empty?
I'm not necessarily saying we should change it, as this would complicate the function and/or return type even more, but it's worth being explicit about.

Reply:
Yeah. I think we'll push it to the caller to verify it's the end so we can alleviate a little bit of complexity on our end. Maybe eventually we can add that functionality and iterators.